### ISSUE TITLE
Tune Hyperparameters

### ISSUE THREAD

**Issue by dharanee99** [dharanee99/DL-FinalProject#7]
Experiment with batch size, learning rate, optimizers, and regularization techniques.
